pom.xml

```diff
@@ -126,7 +126,7 @@ flexible messaging model and an intuitive client API.</description>
     <protobuf3.version>3.11.4</protobuf3.version>
     <protoc3.version>3.11.4</protoc3.version>
     <grpc.version>1.31.0</grpc.version>
-    <protoc-gen-grpc-java.version>1.12.0</protoc-gen-grpc-java.version>
+    <protoc-gen-grpc-java.version>${grpc.version}</protoc-gen-grpc-java.version>
```

Comment:
Is this change related? If not, I'd suggest removing this change from this pull request. Especially we can the way how we define the dependency to using `grpc-bom`. It should be done in a separate PR.

Reply:
yes, this is completely unrelated. I was running the experiment in this PR to see if I could fix the grpc upgrade. I'll remove the changes once #8363 has been merged.